docs/user-guide/rules.md

```diff
@@ -130,6 +130,7 @@ Here are all the rules within stylelint, grouped by the [*thing*](http://apps.wo
 
 ### Selector
 
+- [`selector-attribute-square-brackets-space-inside`](../../src/rules/selector-attribute-square-brackets-space-inside/README.md): Require a single space or disallow whitespace on the inside of the square brackets within selectors.
```

Comment:
"Require a single space or disallow whitespace on the inside of the brackets within attribute selectors."